croissant-rdf/src/croissant_rdf/croissant_harvester.py

```diff
@@ -0,0 +1,275 @@
+import argparse
+import json
+import time
+from abc import ABC, abstractmethod
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from typing import Dict, List, Optional, Union
+
+import requests
+from rdflib import Graph, URIRef
+from rich.progress import track
+
+from croissant_rdf.utils import chunk_data, logger
```

Comment:
Note: in mlcroissant we followed the JAX convention to organize the folders. I found it neat as it clearly indicates which objects are private and public. If you added your scripts within mlcroissant, you'd also inherit from this structure.

```
mlcroissant
|_ __init__.py  # the public API
|_ _src  # all internals
```

Reply:
I am in support of this, but it will take more work and I'm a bit short on time. Hopefully we can find someone who might volunteer.

Reply:
> I am in support of this, but it will take more work and I'm a bit short on time. Hopefully we can find someone who might volunteer.

I could take this over 👍